target/linux/layerscape/image/armv7.mk

```diff
@@ -16,7 +16,8 @@ define Device/Default
 endef
 
 define Device/ls1021atwr
-  DEVICE_TITLE := LS1021ATWR
+  DEVICE_VENDOR := NXP
+  DEVICE_MODEL := LS1021ATWR
```

Comment:
Since model is the same as for the following, we need a variant here ...
Either just use "Default" there, or we have to find out what this device is booting from.
Despite, I just believe you about the vendor here ...

Reply:
https://github.com/openwrt/openwrt/commit/f0ec7bd27d206ea0eb9587e178ce0d7ddc006bce

Looks like "TWR-LS1021" is the better device_model here.

Reply:
> Looks like "TWR-LS1021" is the better device_model here.
TWR-LS1021A!

Oh no ... Why do they do that... Add the prefix as suffix...
So, this would mean that we would have to do research on each of the names, as e.g. LS1043ARDB might then actually be RBD-LS1043A or whatever ...

In your case, I would really think about whether you really want to clean this up or rather keep the current names as already implemented.

Reply:
The RDB (Reference Development Board) seems to be applied as suffix: https://www.nxp.com/design/qoriq-developer-resources/qoriq-ls1012a-development-board:LS1012A-RDB